python/ray/air/result.py

```diff
@@ -21,12 +23,21 @@ class Result:
     Args:
         metrics: The final metrics as reported by an Trainable.
         checkpoint: The final checkpoint of the Trainable.
+        best_checkpoint: The best checkpoint of the Trainable, as
+            determined by the ``checkpointing`` argument of ``RunConfig``,
+            or, if that's unset, by ``metric`` and ``mode`` arguments of
+            ``TuneConfig``.
+            If either of those has not been set, this will be None.
+            May be the same object as ``checkpoint``.
         error: The execution error of the Trainable run, if the trial finishes in error.
+        dataframe: The full result dataframe of the Trainable.
```

Comment:
What does this dataframe contain? Can we clarify that in the docstring? Are the rows ordered by iteration/timestep?

Reply:
It's the same dataframe as the experiment analysis one. The docstring there is also pretty barebones, but I can improve it here. It contains the reported metrics for each iteration of the trial.